src/hypercall.rs

```diff
@@ -21,11 +25,12 @@ use crate::{
 ///
 /// - The return value is only valid, as long as the guest is halted.
 /// - This fn must not be called multiple times on the same data, to avoid creating mutable aliasing.
-pub unsafe fn address_to_hypercall(
+pub unsafe fn address_to_hypercall_v1(
```

Comment:
Should this be moved to a separate file or `uhyve_interface` somehow?

Reply:
Hmm, I think it is fine here